.gitignore

```diff
@@ -3,3 +3,4 @@ dist/
 coverage/
 .directory
 /vscode.json
+.DS_Store
```

Comment:
Do you mind putting this in a [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)?

Reply:
Sure!